src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyndto/BeregningBarnetilsynRequest.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.familie.ef.sak.beregning
+
+import java.math.BigDecimal
+import java.time.YearMonth
+import java.util.UUID
+
+data class BeregningBarnetilsynRequest(
+        val utgiftsperioder: List<UtgiftsperiodeDto>,
+        val kontantstøtteperioder: List<KontantstøttePeriodeDto>,
+        val tilleggsstønadsperioder: List<TilleggsstønadPeriodeDto>
+)
+
+data class TilleggsstønadPeriodeDto(
+        val årMånedFra: YearMonth,
+        val årMånedTil: YearMonth,
+        val beløp: BigDecimal
```

Comment:
Bruker vi egentlige decimaler når vi sender inn disse beløpene? Eller burde vi bare bruke Int? 
Tenker det gjelder for vedtak og 👀

Reply:
Hehe - den har vi snakket mye om. Vi hadde litt lyst til å bruke int, men det blir kanskje riktigere å runde opp etter at vi er ferdige med å regne. Veldig mye hazzle som kanskje ikke er nødvendig, men ... 

Har litt lyst til å ta en prat til om dette med funksjonelle. Vi kom ikke helt i mål sist.

Reply:
Resultatet (beløpene) av en beregning burde sikkert være int.